src/loaddb/load_session.cpp

```diff
@@ -242,13 +248,68 @@ namespace cubload
 
 	// notify session that batch is done
 	m_session.notify_batch_done (m_batch.get_id ());
+
+	thread_ref.m_loaddb_driver->clear ();
```

Comment:
Use local driver variable and also I think it will be better to clear the driver before notifying other threads